Movie_App.js

```diff
@@ -0,0 +1,128 @@
+document.addEventListener("DOMContentLoaded", () => {
```

Comment:
Overall very well done! App's functionalities are as asked from the rubric so is the overall CSS work. (One very minor thing being the logo color (skyblue). I left some feedbacks below but nothing major to fix or to bee re-worked on. Good work and looking forward to more projects like this in the future.